### PR TITLE
fix(icons): arcified `book-*` icons

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?
- [x] Other: Icon update

### Description
Arcified book icons.

## Before Submitting
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/book-dashed.json [hunk 1]
```diff
@@ -23,8 +23,5 @@
   ],
   "categories": [
     "development"
-  ],
-  "aliases": [
-    "book-template"
```
Comment (karsa-mistmere) [inline icons/book-dashed.json:28]: Why?

File: icons/book-copy.svg
Comment (karsa-mistmere) [inline icons/book-copy.svg:1]: The book in this one doesn't quite feel like a book anymore. 🤔

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/book-copy.svg
Comment (jguddas) [reply icons/book-copy.svg:1]: I agree, do you have a suggestion?

File: icons/book-dashed.json (hunk 1 above)
Comment (jguddas) [reply icons/book-dashed.json]: Oups

File: icons/book-copy.svg [hunk 2]
```diff
@@ -10,6 +10,6 @@
   stroke-linejoin="round"
 >
   <path d="M2 16V4a2 2 0 0 1 2-2h11" />
+  <path d="M22 18H11a2 2 0 1 0 0 4h10.5a.5.5 0 0 0 .5-.5V7a.5.5 0 0 0-.5-.5H11a2 2 0 0 0-2 2v12" />
```
Comment (karsa-mistmere) [inline icons/book-copy.svg]: This became .5px shorter than before, making of off-grid.

**Review by karsa-mistmere — APPROVED** [review]

**Review by ericfennis — APPROVED** [review]
Nice!

**Pull request merged**